### DISCUSSION TITLE
Failure counters in RMA memory region permission violations

### DISCUSSION THREAD

**Discussion by traxys** [ofiwg/libfabric#9571] [Q&A]
It's not very clear from the documentation if target failure counters should be incremented when a remote write is received on a memory region lacking FI_REMOTE_WRITE, or a remote read on a memory region lacking FI_REMOTE_READ.

What would be the correct behaviour ?

**Comment by j-xiong** [comment]
The corresponding error counter (that has been bound to the endpoint with the correct flag) should be incremented when an incoming RMA op fails.
